src/main.rs

```diff
@@ -24,7 +28,7 @@ fn main() {
 
     let opengl = OpenGL::V3_2;
 
-    let mut window: PistonWindow =
+    let mut window: GlutinWindow =
```

Comment:
I don't think it limits us in any way. I haven't heard of `PistonWindow` automatically picking a backend. I think it uses `GlutinWindow` by default.